SConstruct

```diff
@@ -2,31 +2,49 @@ env = Environment()
 
 ## dependencies ##
 
-yaml_cpp_env = env.Clone()
-yaml_cpp_env.VariantDir('_build/yaml-cpp', 'lib/yaml-cpp', duplicate=0)
-yaml_cpp_env.Command('_build/yaml-cpp/libyaml-cpp.a', '', 'cd _build/yaml-cpp && cmake ../../lib/yaml-cpp && make')
+yamlcpp_env = env.Clone()
+yamlcpp_env.VariantDir('_build/yaml-cpp', 'lib/yaml-cpp', duplicate=0)
+yamlcpp_env.Command('_build/yaml-cpp/libyaml-cpp.a', '',
+                    'cp -r lib/yaml-cpp _build/ && ' + \
+                    'cd _build/yaml-cpp && cmake . && make')
+
+libptpll_env = env.Clone()
+libptpll_env.VariantDir("_build/libptpll", "lib/libptpll", duplicate=0)
+libptpll_env.Command('_build/libptpll/libptpll_static.a', '',
+                     'cp -r lib/libptpll _build/ && cd _build/libptpll && ' + \
+                     'make && cp -t . _build/src/libptpll_static.a ' + \
+                     '_build/lib/_prefix/lib/libpll_algorithm.a ' + \
+                     '_build/lib/_prefix/lib/libpll_optimize.a ' + \
+                     '_build/lib/_prefix/lib/libpll_tree.a ' + \
+                     '_build/lib/_prefix/lib/libpll_util.a ' + \
+                     '_build/lib/_prefix/lib/libpll.a ' + \
+                     '_build/lib/lesplace/src/liblesplace-static.a')
```

Comment:
linearham manually builds yaml-cpp and libptpll (+ its dependencies).  Our tree structure is directly from libptpll.